samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj

```diff
@@ -68,6 +62,7 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
+	  <PackageReference Include="Azure.Storage.Blobs" Version="12.8.1" />
```

Comment:
This reference looks out of place, like it was added with .NET Core rather than a legacy .NET Framework tool.   We should probably continue to use the old format or upgrade everything.

Reply:
I have updated this reference with the old format.